CodeChat_Server/CodeChat_Server/templates/sphinx/conf.py

```diff
@@ -18,7 +18,7 @@
 # *********************************************************************
 # |docname| - Template configuration file for a Sphinx CodeChat project
 # *********************************************************************
-# This file configures Sphinx, which transforms restructured text (reST) into
+# 40, which transforms restructured text (reST) into
```

Comment:
??? Revert this change.

Reply:
fixed

Reply:
The fix for this and the next comment are in a new PR. Not sure if this is what you wanted ( I remember in your lecture you said to make small PRs but not sure if this is the case you wanted it for). If not, just so I know in the future, is it possible for another user to add changes to a branch they have checked out and have it show up in the same PR?

Reply:
Sorry, missed this until now. You can always makes changes to an existing PR by pushing more changes to it.